pipeline_tools/shared/submission/create_analysis_metadata.py

```diff
@@ -192,12 +205,17 @@ def create_analysis_files(
             ),
             'schema_type': 'file',
             'provenance': {
-                'document_id': get_uuid5(output['sha256']),
+                'document_id': get_uuid5(
+                    str(project_id) + get_relative_file_location(output['file_path'])
+                ),
                 'submission_date': convert_datetime(output['timestamp']),
             },
             'file_core': {
                 'file_name': output['file_path'].split('/')[-1],
                 'format': get_file_format(output['file_path'], extension_to_format),
+                'content_description': DCP2_MATRIX_CONTENT_DESCRIPTION
+                if output['file_path'].endswith(".loom")
+                else [],
```

Comment:
Just to make sure the type does not matter here, i.e. when `DCP2_MATRIX_CONTENT_DESCRIPTION` is not given, an empty list `[]` instead of empty map `{}` should be used here. Asking since `DCP2_MATRIX_CONTENT_DESCRIPTION` is a map/dict.

Reply:
Thank you, @rexwangcc! I actually noticed this today when I went to validate the output. In all cases, the result should be an array, rather than a single object. Making the change now.

Reply:
Cool!